src/tree_graphviz.ml

```diff
@@ -73,18 +79,18 @@ let to_dot : string -> sym -> unit = fun fname s ->
     let pp_dotterm : dot_term pp = fun oc dh ->
       let out fmt = Format.fprintf oc fmt in
       match dh with
-      | DotAbst(v)           -> out "λ%a" Print.pp_tvar v
+      | DotAbst(v,id)        -> out "λ%s%d" (Bindlib.name_of v) id
```

Comment:
Why not `out "λ%s%d" Tree.var_prefix i` like in the case `DotCons(Vari(i))`?

Reply:
It should be rigorously the same